starfish/image/_stack.py

```diff
@@ -155,15 +156,92 @@ def _build_slice_list(
 
         return tuple(slice_list), axes
 
+    def _iter_indices(self, is_3d: bool=False) -> Iterator[Mapping[Indices, int]]:
```

Comment:
I like this much more, although I think someone suggested is_3d instead of that. Are we ok with `is_volume` despite some of our image stacks representing focal volumes, not true tissue volumes?